src/components/metrics/card/index.js

```diff
@@ -178,10 +190,26 @@ class MetricCard extends Component {
     return errors ? errors.filter(e => !!e) : []
   }
 
+  showSimulation() {
+    const stats = _.get(this.props, 'metric.simulation.stats')
+    if (stats && _.isFinite(stats.mean) && _.isFinite(stats.stdev) && _.isFinite(stats.length)) {
```

Comment:
Why do you check the mean and the length here as well? Could this whole function be simplified to:
```
const stdev = _.get(this.props, 'metric.simulation.stats.stdev')
return !!stdev && stdev !== 0
```
?